lib/dpul_collections/indexing_pipeline/figgy/hydration_consumer.ex

```diff
@@ -94,8 +96,32 @@ defmodule DpulCollections.IndexingPipeline.Figgy.HydrationConsumer do
     |> Broadway.Message.put_batcher(:noop)
   end
 
+  def extract_related_data(resource) do
```

Comment:
Part of me feels like all this code should be somewhere else, but I don't really know where. Like I'm scared it's gonna get a lot bigger, but maybe it won't?